packages/dataviews/src/validation.ts

```diff
@@ -32,6 +32,7 @@ export function isItemValid< Item >(
 			if (
 				( field.type === 'text' && isEmptyNullOrUndefined( value ) ) ||
 				( field.type === 'email' && isEmptyNullOrUndefined( value ) ) ||
+				( field.type === 'url' && isEmptyNullOrUndefined( value ) ) ||
```

Comment:
I just realized we're missing `telephone` here (note the save button is enabled but should be disabled):

https://github.com/user-attachments/assets/950a8ab6-9d56-4ced-beb1-c72eb4ef0239

Reply:
Good catch. I updated.